packages/wrangler/src/__tests__/dev.test.ts

```diff
@@ -142,6 +142,7 @@ describe.sequential("wrangler dev", () => {
 		msw.resetHandlers();
 		spy.mockClear();
 		setSpy.mockClear();
+		logger.loggerLevel = "log";
```

Comment:
The logger implementation seems to treat the setter as an override. So this won't be able to restore the default behavior which used `env` and fallback to `log`:

https://github.com/cloudflare/workers-sdk/blob/35c1dc32137f6440c3e514fdc54b5f471fdff7aa/packages/wrangler/src/logger.ts#L64-L66

Reply:
Looks like you could use `logger.resetLoggerLevel()`